Stolovka/Packages/SmokersStolovka/Sources/SmokersStolovka/MainMenu/Managers/ShoppingCartManager.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ShoppingCartManager.swift
+//  
+//
+//  Created by Nikita Shvad on 18.02.2022.
+//
+
+import Foundation
+import UIKit
+
+protocol ShoppingCartManagerProtocol {
+    
+    func addToCart(foodItem: FoodItem, completion: @escaping (Result<Void, Error>) -> Void)
+    func removeFromCart(foodItem: FoodItem, completion: @escaping (Result<Void, Error>) -> Void)
+    func fetchCartItems(completion: @escaping (Result<Void, Error>) -> Void)
```

Comment:
Here instead of `Void` should be `[FoodItem]`
```suggestion
    func fetchCartItems(completion: @escaping (Result<[FoodItem], Error>) -> Void)
```